lib/google_maps_geocoder/google_maps_geocoder_error_handler.rb

```diff
@@ -0,0 +1,17 @@
+# Handling errors form Google
+module GoogleMapsGeocoderErrorHandler
+  # Parent class for more concrete errors
+  class GeneralGoecodingError < StandardError
+    def initialize(response_json = '')
+      @json = response_json
+      super
+    end
+
+    def message
+      "Google returned:\n#{@json.inspect}"
+    end
+  end
```

Comment:
Ah, now I see why you wanted the superclass. I'm good with this, but please shorten `GeneralGoecodingError` to `GeocodingError`.

Also, is the module necessary? Can't we have the error classes in the same level and file as `GoogleMapsGeocoder`?